File: aemet/Aemet-Java-IvanRC/src/main/java/dto/InformesDto.java
package dto;

import com.squareup.moshi.Json;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Json(name = "informes")
@Root(name = "informes")
public class InformesDto {

    @ElementList(name = "informes", inline = true)
    List<MapaRegistroDto> informes;
}

File: aemet/Aemet-kotlin-IvanRC/src/main/kotlin/dto/InformesDto.kt
package dto

import com.squareup.moshi.Json
import org.simpleframework.xml.ElementList
import org.simpleframework.xml.Root

@Json(name = "mapa_informes")
@Root(name = "mapa_informes")
data class InformesDto(
    @Json(name = "mapa_informes")
    @field:ElementList(name = "mapa_informes", inline = true)
    @param:ElementList(name = "mapa_informes", inline = true)
    val conjuntoInformesDto: List<MapaRegistrosDto>
)